src/video/ffmpeg/ffmpeg_common.h

```diff
@@ -21,6 +21,7 @@
 extern "C" {
 #endif
 #include <libavcodec/avcodec.h>
+#include <libavcodec/bsf.h>
```

Comment:
make it alphabetical?

Reply:
no, that would add diff noise, so it belongs to a separate commit

```diff
 #include <libavcodec/avcodec.h>
 #include <libavcodec/bsf.h>
-#include <libavformat/avformat.h>
-#include <libavformat/avio.h>
 #include <libavfilter/avfilter.h>
 #include <libavfilter/buffersink.h>
 #include <libavfilter/buffersrc.h>
+#include <libavformat/avformat.h>
+#include <libavformat/avio.h>
 #include <libavutil/avutil.h>
-#include <libavutil/pixfmt.h>
-#include <libavutil/opt.h>
-#include <libavutil/version.h>
 #include <libavutil/dict.h>
 #include <libavutil/display.h>
 #include <libavutil/file.h>
+#include <libavutil/opt.h>
+#include <libavutil/pixfmt.h>
+#include <libavutil/version.h>
 #include <libswresample/swresample.h>

```